tests/requirements.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+#- src: openwisp.openwisp2
+
+- src: https://github.com/openwisp/ansible-openwisp2
+  version: master
+  name: openwisp.openwisp2
```

Comment:
ensure files always end with a new line (set your editor to do so automatically)